src/com/company/reverse/Reverse.java

```diff
@@ -0,0 +1,17 @@
+package com.company.reverse;
+
+
+public class Reverse {
+    private static final String MATCH_ALL_DIGITS = "[\\d+]";
+    private static final String STRING_REGEX = "";
+
+    public String ActionOfReversing(String stringToReverse) {
```

Comment:
Java style guide suggests different naming strategy for methods. Please see [https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names](url)

Reply:
Done